Jellyfin.Api/Controllers/DynamicHlsController.cs

```diff
@@ -1809,7 +1810,7 @@ private string GetVideoArguments(StreamState state, int startNumber, bool isEven
             || string.Equals(codec, "hevc", StringComparison.OrdinalIgnoreCase))
         {
             if (EncodingHelper.IsCopyCodec(codec)
-                && (string.Equals(state.VideoStream.VideoRangeType, "DOVI", StringComparison.OrdinalIgnoreCase)
+                && (state.VideoStream.VideoRangeType.Equals(VideoRangeType.DOVI)
```

Comment:
```suggestion
                && (state.VideoStream.VideoRangeType == VideoRangeType.DOVI
```
For every enum comparison.

Reply:
Is this just style preference oris there more to it?

Reply:
The `.Equals` operator requires [boxing](https://learn.microsoft.com/en-us/dotnet/csharp/programming-guide/types/boxing-and-unboxing) to convert to an object, which is less efficient then directly comparing